Assignment1_q2_Singly_Linked_List.py

```diff
@@ -0,0 +1,57 @@
+from Assignment1_q2_Node import *
+
+# My implementation of Singly Linked List
+class SinglyLinkedList(object):
+
+    def __init__(self, head = None):
+        self.head = head
+
+    # Inserting elements to the front of the list
+    def insert_first(self, data):
+        new_node = Node(data)
+        new_node.set_next_node(self.head)
+        self.head = new_node
+
+    # Get kth to the last element of the list - assignment question
+    # (The idea is to have two pointers, both set to head at the start
+    # we move the first pointer k times - and thus it has len(list) - k
+    # elements till the end of the list - and then start moving both pointers
+    # until the first pointer reaches the end of the list, then return the second pointer)
+    def get_from_last(self, k):
+        main_pointer = self.head
+        helper_pointer = self.head
+
+        if self.head is None:
+            raise ValueError("Empty list!")
```

Comment:
it returns IndexError now but I left ValueError for k out of bounds, is that okay?